### DISCUSSION TITLE
Information available during evaluation

### DISCUSSION THREAD

**Discussion by rajas1310** [empriselab/RCareWorld#114] [Other Q&A] [answered]
Hi,
What information regarding the state of the objects (sponge, drawer, cloth points, human) will be exposed to us during the evaluation?
Also is there a limit on the number of cameras we can use?
I need to know this so that I don't design an algorithm that operates on unavailable data.

**Comment by YoruCathy** [comment, marked as answer]
Hi, there's no restriction on what information you can use as long as you can get it from the simulation. There are also no limits on the camera numbers. However, we don't suggest using too many cameras (more than 5) since that might make the simulation run very slow.
